src/services/xpService.js

```diff
@@ -133,7 +126,6 @@ exports.processDecrementXpScript = function() {
 
     if(exports.shouldDecrementXp(daysSinceContact, data[userId].penaltyCount)) {
       let decrementedXp = calculateDecrementedXp()
-      console.log(`INFO ONLY: Decrementing XP for user ${data[userId].username} from ${data[userId].currentXp} to ${decrementedXp}...`)
```

Comment:
Lets leave this one in for now (but remove 'INFO ONLY:') just so if something goes south I have a log of why.